packages/grpc-native-core/src/grpc_extension.js

```diff
@@ -31,17 +31,27 @@ var binding;
 try {
   binding = require(binding_path);
 } catch (e) {
-  var fs = require('fs');
-  var searchPath = path.dirname(path.dirname(binding_path));
-  var searchName = path.basename(path.dirname(binding_path));
-  var foundNames = fs.readdirSync(searchPath);
+  let fs = require('fs');
+  let searchPath = path.dirname(path.dirname(binding_path));
+  let searchName = path.basename(path.dirname(binding_path));
+  let foundNames;
+  try {
+    foundNames = fs.readdirSync(searchPath);
+  } catch (readDirError) {
+    let message = `The gRPC binary module was not installed
+This may be fixed by running "npm rebuild"
+Original error: ${e.message}`;
```

Comment:
Should this be `readDirError.message`? (And same with `e.code` below)

Reply:
Also, it seems like the first two lines can fit together on a single line.

Reply:
No, it's supposed to be `e.message` and `e.code`, because `e` is the original error that we are trying to communicate.

Also, I just made the structure of the error message match the other error. I could merge those lines, though.

Reply:
OK, I merged the lines.

Reply:
Right, didn't see that 🙃